ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldProcess.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "ThermoMechanicalPhaseFieldProcess-impl.h"
+#include "ThermoMechanicalPhaseFieldProcess.h"
```

Comment:
Move `#include "ThermoMechanicalPhaseFieldProcess.h"` to the line before `#include "ThermoMechanicalPhaseFieldProcess-impl.h"`

Reply:
done.